plugin/pkg/admission/resourcequota/controller.go

```diff
@@ -118,6 +124,7 @@ func newQuotaEvaluator(client clientset.Interface, registry quota.Registry) (*qu
 		registry:        registry,
 		liveLookupCache: liveLookupCache,
 		liveTTL:         time.Duration(30 * time.Second),
+		updatedQuotas:   map[string]*api.ResourceQuota{},
```

Comment:
I see check cache will delete caches on subsequent calls to that namespace, but I have not yet reasoned through if that is sufficient....